src/RegistrationController.js

```diff
@@ -128,6 +128,13 @@ function (
             form.add(SubSchema.extend({id: 'subschemas-' + name, subSchemas: subSchemas}));
           }
         });
+        // fire password validation when username changes
+        this.listenTo(this.model, 'change:userName', function() {
+          if (this.model.get('userName') && this.model.get('password')) {
+            var passwordField = Okta.$('.registration .okta-form-input-field input[type="password"]');
```

Comment:
Is there a way for the password field to listen to the "change:userName" event?
Also the get rid of the selector too.

Reply:
I would have to try that out. But that would move the logic back to RegistrationFormFactory.js?

Reply:
That does seem to work, and its a much smaller change. I can revert this and move it back to the RegistrationFormFactory.js